packages/wonder-stuff-server-google/src/start-server.js

```diff
@@ -0,0 +1,78 @@
+// @flow
```

Comment:
This is the remainder of `shared/start-gateway.js` from render-gateway - it does the bits that are Google-specific, then delegates to the `start-server.js` that we extracted in the last PR.